src/simpleNbox.cpp

```diff
@@ -998,4 +1132,201 @@ void SimpleNbox::set_c0(double newc0)
         
 }
 
+// Add a biome to a time-series map variable (e.g. veg_c_tv)
+template <class T_data>
+void SimpleNbox::add_biome_to_ts(tvector<std::map<std::string, T_data>>& ts,
+                                 const std::string& biome,
+                                 T_data init_value) {
+    // First, check if a biome of this name already exists in the data
+    if ( ts.get(ts.firstdate()).count( biome ) ) {
+        H_THROW( "Biome '" + biome + "' already exists in data." );
+    }
+    
+    // Loop over time steps, and set the variable to the provided `init_value`
+    std::map<std::string, T_data> newval;
+    for ( double i = ts.firstdate(); i < ts.lastdate(); i++ ) {
+        if (ts.exists(i)) {
+            newval = ts.get(i);
+            newval[ biome ] = init_value;
+            ts.set(i, newval);
+        }
+    }
+}
+
+// Remove a biome from a time-series map variable
+template <class T_map>
+void SimpleNbox::remove_biome_from_ts(tvector<T_map>& ts,
+                                      const std::string& biome) {
+    // We don't need to check for presence of `biome` here because the
+    // `<std::map>.erase()` method is effectively a no-op when given a
+    // non-existent key.
+    T_map currval;
+    for ( double i = ts.firstdate(); i < ts.lastdate(); i++ ) {
+        if (ts.exists(i)) {
+            currval = ts.get(i);
+            currval.erase(biome);
+            ts.set(i, currval);
+        }
+    }
+}
+
+// Remove a biome from a time-series map variable
```

Comment:
Comment is inaccurate; this is the rename function, not the remove function.
```suggestion
// Rename a biome in a time-series map variable
```

Reply:
Good catch! Addressed in 53de23c293e781409873d68da2e3200b25a80664.